content/zh-cn/dogepedia/articles/what-is-dogecoin.md

```diff
@@ -1,21 +1,21 @@
 +++
-title = "What is Dogecoin?"
-date = "2021-10-22"
+title = "狗狗币是什么?"
+date = "2022-08-07"
 type = "article"
 
 [ author ]
-  name = "Dogecoin"
+  name = "Tiger Hu"
```

Comment:
please keep the original author to 'Dogecoin' as it's an official written article. 

many thanks.